AppCenter/AppCenter/Internals/Model/MSLogConversion.h

```diff
@@ -12,6 +12,6 @@
  *
  * @return An array of MCSCommonSchemaLog objects.
  */
-- (NSArray<MSCommonSchemaLog *> *)toCommonSchemaLogsWithFlags:(int64_t)flags;
+- (NSArray<MSCommonSchemaLog *> *)toCommonSchemaLogsWithFlags:(NSUInteger)flags;
```

Comment:
Can't we use MSFlags everywhere? The next file in the review list uses it in saveLog.